__tests__/server.test.js

```diff
@@ -232,17 +232,17 @@ describe.skip("/api/workouts/:level", () => {
                 })
             })
         })
-        test("400: Responds with a bad request message if ID is invalid", () => {
+        test.skip("400: Responds with a bad request message if workout level is invalid", () => {
             return request(app)
-            .get("/api/workouts/out_id")
+            .get("/api/workouts/wrongLvl")
             .expect(400)
             .then((response) => {
-                expect(response.body.message).toBe("Invalid ID")
+                expect(response.body.message).toBe("Invalid query")
```

Comment:
I think "Invalid level" might be better here. Technically, it's not quite a query, but rather, a parametric endpoint, so it would be "Invalid parameter", but I think it's fine being a little more specific here, saying "Invalid level"